packages/MJServer/src/resolvers/AskSkipResolver.ts

```diff
@@ -378,9 +378,10 @@ export class AskSkipResolver {
   protected BuildSkipEntities(): SkipEntityInfo[] {
     // build the entity info for skip in its format which is
     // narrower in scope than our native MJ metadata
-    // don't pass the mj_core_schema entities
+    // don't pass the mj_core_schema entities by default, but allow flexibilty 
+    // to include specific entities from the MJAPI config.json
     const md = new Metadata();
-    return md.Entities.filter((e) => e.SchemaName !== mj_core_schema).map((e) => {
+    return md.Entities.filter((e) => e.SchemaName !== mj_core_schema || configInfo.askSkip.entitiesToSendSkip.includeEntitiesFromExcludedSchemas.includes(e.Name)).map((e) => {
```

Comment:
@nico-oz  this is the right logic but assumes people don't have white space _within_ their strings in the JSON file and/or don't have the wrong **case** for any individual letter. So what you should do is prob create a local array say it is called `includedEntities` and use the `map` function on the `configInfo.askSkip.entitiesToSendSkip.includeEntitiesFromExcludedSchemas` to transform each element by trim() and toLowerCase() into this new local array copy. Then use that and also convert the entity name from the `e` variable inside the arrow function to `trim().toLowerCase()` as well so both sides are adjusted properly. 

That will catch any sloppy config files. Rest of this looks good!